content/zh/docs/reference/glossary/kubernetes-api.md

```diff
@@ -17,7 +17,7 @@ tags:
 title: Kubernetes API
 id: kubernetes-api
 date: 2018-04-12
-full_link: /zh/docs/concepts/overview/kubernetes-api/
+full_link: /docs/concepts/overview/kubernetes-api/
```

Comment:
Why this change?

Reply:
It is the original English version, commented out.

Reply:
It is the original English version, commented out. @sftim